lib/checks/mobile/meta-viewport.json

```diff
@@ -8,7 +8,7 @@
     "impact": "critical",
     "messages": {
       "pass": "<meta> tag does not disable zooming on mobile devices",
-      "fail": "<meta> tag disables zooming on mobile devices"
+      "fail": "{{~it.data:value}} {{=value}}{{~}} on <meta> tag disables zooming on mobile devices"
```

Comment:
data seems to be an array, values should be joined here to construct the string...

Reply:
Changed it to a string.

Reply:
We need to do `{{=it.data}}` otherwise you get this:

![screenshot 2018-08-09 11 18 22](https://user-images.githubusercontent.com/530687/43890102-1c3d8d0e-9bc6-11e8-9552-50cb94f58950.png)